<launch> 
    <node name="g3t1_2" pkg="component" type="g3t1_2" output="screen" />

    <param name="frequency" value="1" /> <!-- 1 Hz  -->
    
    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- markov chain -->
    <param name="state0" value="10,10,10,20,40" />
    <param name="state1" value="10,10,10,20,40" />
    <param name="state2" value="10,10,10,20,40" />
    <param name="state3" value="10,10,10,20,40" />
    <param name="state4" value="10,10,10,20,40" />

    <!-- risk values -->
    <param name="HighRisk0" value="0,70" />
    <param name="MidRisk0" value="70,85" />
    <param name="LowRisk" value="85,97" />
    <param name="MidRisk1" value="97,115" />
    <param name="HighRisk1" value="115,300" />

    <!-- accuracy in percentage -->
    <param name="data_accuracy" value="90" type="double" />
    <param name="comm_accuracy" value="100" type="double" />

    <!-- persistence -->
    <param name="persist" value="true" type="bool" />
    <param name="path" value="g3t1_2_output.csv" type="str" />

</launch>